### ISSUE TITLE
Publishing not starting on staging

### ISSUE THREAD

**Issue by karenc** [Connexions/cnx-deploy#360]
## Summary

Publishing fails to start on staging.  Here is the traceback on prod08 in `/var/log/supervisor/publishing0-stdout---supervisor-_n91gf.log`:

```
Traceback (most recent call last):
  File "/var/cnx/venvs/publishing/bin/pserve", line 11, in <module>
    sys.exit(main())
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 58, in main
    return command.run()
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 328, in run
    global_conf=vars)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 363, in loadapp
    return loadapp(app_spec, name=name, relative_to=relative_to, **kw)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 247, in loadapp
    return loadobj(APP, uri, name=name, **kw)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 272, in loadobj
    return context.create()
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 710, in create
    return self.object_type.invoke(self)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 229, in invoke
    filtered = context.next_context.create()
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 710, in create
    return self.object_type.invoke(self)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 146, in invoke
    return fix_call(context.object, context.global_conf, **context.local_conf)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/paste/deploy/util.py", line 55, in fix_call
    val = callable(*args, **kw)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/cnxpublishing/main.py", line 102, in main
    config = Configurator(settings=settings, root_factory=RootFactory)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/config/__init__.py", line 330, in __init__
    exceptionresponse_view=exceptionresponse_view,
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/config/__init__.py", line 445, in setup_registry
    self.include(inc)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/config/__init__.py", line 769, in include
    c = self.maybe_dotted(callable)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/config/__init__.py", line 869, in maybe_dotted
    return self.name_resolver.maybe_resolve(dotted)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/path.py", line 320, in maybe_resolve
    return self._resolve(dotted, package)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/path.py", line 327, in _resolve
    return self._zope_dottedname_style(dotted, package)
  File "/var/cnx/venvs/publishing/local/lib/python2.7/site-packages/pyramid/path.py", line 376, in _zope_dottedname_style
    found = __import__(used)
ImportError: No module named pyramid_sawing
```

## Environment

staging (most likely also on prod)

**Comment by karenc** [comment]
https://github.com/Connexions/cnx-deploy/commit/d9d2278bf09066c25e3af9bf1d9a14eaa030a434

**Comment by karenc** [comment]
The pull request for installing pyramid_sawing on staging has been merged in #363.
